plato-api/src/main/java/org/techpleiad/plato/api/response/ServicesAcrossBranchValidateResponseTO.java

```diff
@@ -12,4 +12,5 @@
 public class ServicesAcrossBranchValidateResponseTO {
     private String service;
     private List<BranchProfileReportResponseTO> report;
+
```

Comment:
Try to eliminate  @such not required commits @DialgicMew

Reply:
yes